src/ReactDOMRe.re

```diff
@@ -41,6 +41,41 @@ let renderToElementWithId = (reactElement, id) =>
   | Some(element) => render(reactElement, element)
   };
 
+module Unstable = {
+  type root;
+
+  [@bs.module "react-dom"]
+  external createRoot: Dom.element => root = "unstable_createRoot";
```

Comment:
```suggestion
  external createRoot: Dom.element => root = "createRoot";
```

Reply:
Just checked and it looks like they've removed unstable_ - but we should make sure our React versions are lining up for this change. https://reactjs.org/docs/concurrent-mode-reference.html#createroot